src/firefly/js/ui/tap/AdvancedADQL.jsx

```diff
@@ -25,18 +26,25 @@ import {
 } from './TapUtil';
 import {getColumnIdx} from '../../tables/TableUtil.js';
 import {dispatchValueChange} from '../../fieldGroup/FieldGroupCntlr.js';
-import West from '@mui/icons-material/West';
+import {useColorMode} from 'firefly/ui/FireflyRoot';
+import CssStrThemeWrapper from 'firefly/ui/CssStrThemeWrapper';
 
+//--- PrismJs imports ---
 import Prism from 'prismjs';
-// bliss is needed for prism-live
-import 'blissfuljs';
-import '../../externalSource/prismLive/prism-sql.js';
+import 'prismjs/components/prism-sql.js';
+import 'blissfuljs'; // bliss is needed for prism-live
 import '../../externalSource/prismLive/prism-live.js';
 
-import '../../externalSource/prismLive/prism.css';
-import '../../externalSource/prismLive/prism-live.css';
+// 'primjs/themes/*.css' uses same classnames, so to avoid css overriding, we import light and dark theme css files
+// into 2 separate strings using `?raw` resourceQuery (see the corresponding rule defined in webpack.config.js)
+import prismLightCss from 'prismjs/themes/prism.css?raw';
+import prismDarkCss from 'prismjs/themes/prism-okaidia.css?raw';
+
+// 'prism-live.css' needs to be imported in same way as above otherwise specificity of CSS selectors will change
+import prismLiveCss from '../../externalSource/prismLive/prism-live.css?raw';
```

Comment:
I removed the inline webpack syntax for raw-loader (also removed from package.json). I noticed that webpack supports raw-loading in-built starting with v5 (which is what we're using).